doc/conf.py

```
@@ -58,10 +58,19 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
-# The short X.Y version.
-version = '0.1'
-# The full version, including alpha/beta/rc tags.
-release = '0.1.0-dev'
+# Read version from VERSION file (single source of truth)
+_version_file = os.path.join(os.path.dirname(__file__), '..', 'VERSION')
+if os.path.exists(_version_file):
+    with open(_version_file, 'r') as f:
```

Ruff (0.14.10)

Unnecessary mode argument

Remove mode argument

(UP015)